std/getopt.d

```diff
@@ -468,13 +467,13 @@ unittest
    string and its bound pointer.
 */
 enum config {
-    /// Turns case sensitivity on
+    /// Turn case sensitivity on
     caseSensitive,
-    /// Turns case sensitivity off
+    /// Turn case sensitivity off
     caseInsensitive,
-    /// Turns bundling on
+    /// Turn bundling on
     bundling,
-    /// Turns bundling off
+    /// Turn bundling off
```

Comment:
Because the grammar was inconsistent. I changed
 
Turns / Pass / Signal / Stop / Do not / Makes
to
Turn / Pass / Signal / Stop / Do not / Make